src/gcodePlanner.cpp

```diff
@@ -619,6 +619,11 @@ void GCodePlanner::writeGCode(GCodeExport& gcode)
     GCodePathConfig* last_extrusion_config = nullptr; // used to check whether we need to insert a TYPE comment in the gcode.
 
     int extruder = gcode.getExtruderNr();
+    bool acceleration_enabled = storage.getSettingBoolean("acceleration_enabled");
+    bool jerk_enabled = storage.getSettingBoolean("jerk_enabled");
+    bool speed_for_pressure_enabled = storage.getSettingBoolean("speed_for_pressure_enabled");
```

Comment:
I would change this so that this is at least configurable per extruder. Enabling or disabling this might heavily depend on the type of extruder you have and the type of material.